src/main/java/core/basesyntax/Paremetres.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Paremetres{
```

Comment:
```suggestion
public interface AreaCalculator{
```

не информативное название интерфейса